desc/vmec_utils.py

```diff
@@ -192,6 +192,7 @@ def fourier_to_zernike(m, n, x_mn, basis):
     return x_lmn
 
 
+# FIXME: this always returns the full double Fourier basis regardless of symmetry
```

Comment:
@ddudt does this need to be fixed imminently?

Reply:
No it does not. In the current code this function only gets called by `VMECIO.save()`, where it is always used in the combination:
```
m, n, x_mn = zernike_to_fourier(eq.R_lmn, basis=eq.R_basis, rho=r_full)
xm, xn, s, c = ptolemy_identity_rev(m, n, x_mn)
```
 So as long as all of the `vmec_util` functions are self-consistent, everything will work fine. But long-term it would be good to generalize them to handle other custom applications.